### PR TITLE
Add knative-installer component

### PR DESCRIPTION
**Pull request by sjanota**
**Description**

Changes proposed in this pull request:

There is new component: knative-installer. It is a kubernetes job which installs knative along with kyma.

**Related issue(s)**

Based on @Abd4llA's #1360. Changes taken from there:
* components/knative-installer/install.sh
* components/knative-installer/is-ready.sh

### CODE REVIEW

File: components/knative-installer/install.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+######### Knative build & serving ########
+
+echo "Installing Knative build and serving ..."
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
+curl -L ${SERVING_URL} \
```
Comment (pbochynski) [inline components/knative-installer/install.sh]: I don't like the idea we download the serving sources during installation. Can't we put it in the docker image? This way we can have quicker and repeatable installations.

Comment (sjanota) [reply components/knative-installer/install.sh]: I think it's reasonable as we depend on knative release structure in scripts anyway. There is no point in having it configurable.

Comment (sjanota) [reply components/knative-installer/install.sh]: Fixed

File: components/knative-installer/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## Knative installer
+
+## Overview
+
+Knative is distributes as single YAML file with resources definitions. `knative-installer` containerizes the process to make it possible to install Knative with Kyma's installer.
```
Comment (tomekpapiernik) [inline components/knative-installer/README.md]:
```suggestion
Knative is distributed as a single YAML file containing resource definitions. The Knative installer containerizes the process to allow the installation of Knative using the Kyma installer.
```

File: components/knative-installer/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+## Prerequisites
+
+Knative requires istio in `istio-system` namespace. It must be installed before `isito-kyma-patch`.
```
Comment (tomekpapiernik) [inline components/knative-installer/README.md]:
```suggestion
Knative requires running Istio in the `istio-system` Namespace. It must be installed before `isito-kyma-patch` is applied.
```

**Pull request merged**